.github/workflows/jira.yaml

```diff
@@ -58,6 +58,21 @@ jobs:
         else
           echo "type=ISS" >> $GITHUB_OUTPUT
         fi
+
+        echo "==> Determining team labels"
+        teams_field='[]'
+        for team in $(echo "${{ inputs.teams-array }}" | jq -r '.[]'); do
+          if [[ "$team" == "applications" ]] || [[ "$team" == "foundations" ]]; then
+            t="${team}-eco"
+            # append team label to array
+            teams_field="$(echo "$teams_field" | jq --arg tt "$t" '. |= . + [$tt]')"
+          else
+            echo "==> Error: unsupported teams-array input: ${{ inputs.teams-array }}"
+            exit 1
```

Comment:
Should we be more lenient?

For example, if my repository directly sets the new proper team name e.g. `application-eco` I think this preprocess step should let me do it.

Reply:
The above comment probably also comes in handy when trying to onboard new repositories onto common workflows, since we won't have to continue with the deprecated `applications` input on new repos and would be able to pass in `applications-eco` from the get-go

Reply:
Yeah, I started to write code to handle the new labels too. But it started to get much more complicated so I just pushed up what I had. I am fine with being more lenient though if thats what we want. In that case, I can make this else block just use the provided `inputs.teams-array` and continue the workflow. Thoughts?

Reply:
Sounds like a good solution, +1 from me.

Reply:
Sounds great, thanks JM!